pandas/_libs/window.pyx

```diff
@@ -1267,8 +1267,8 @@ cdef _roll_min_max(ndarray[numeric] values, int64_t win, int64_t minp,
 
 
 cdef _roll_min_max_variable(ndarray[numeric] values,
-                            ndarray[int64_t] starti,
-                            ndarray[int64_t] endi,
+                            int64_t[:] starti,
+                            int64_t[:] endi,
```

Comment:
can you add a `const` modifier to these? (and below)

Reply:
I added const to this one, but when I was verifying the args usage in the below function, I saw that both args weren't even being used at all!  So those have been removed.